### DISCUSSION TITLE
[Prettier Plugin] Would like to add support for tailwind-styled-components

### DISCUSSION THREAD

**Discussion by guramnozadze** [tailwindlabs/tailwindcss#11703] [Ideas]
```
// styles.ts
import tw from 'tailwind-styled-components'

export const FooterBox = tw.div'flex justify-between mt-20 flex-wrap'
```
Currently it doesn't format this file.

prettier-plugin-tailwindcss is great byt doesn't format tailwind-styled-components, how can I add it?

**Comment by Whearclas34** [comment]
First, make sure you have the required packages installed. Install prettier, prettier-plugin-tailwind, and tailwind-styled-components if you haven't already.
To add support for tailwind-styled-components in the Prettier plugin, you can try adding a custom configuration to the plugin. Check the plugin's documentation for any available options or extensions specifically for tailwind-styled-components. If there are none, you may need to consider creating a custom formatter or exploring alternative plugins that support this specific styling library.
Keep in mind that the support for formatting specific libraries or syntax might vary based on available plugins and configurations. Be sure to refer to the documentation of the relevant packages for any updates or additional details in [Pickleball Paddle on a Plane?](https://www.ballsportspro.com/can-you-bring-a-pickleball-paddle-on-a-plane/)
